public/apps/configuration/panels/role-edit/role-edit.tsx

```diff
@@ -180,12 +187,19 @@ export function RoleEdit(props: RoleEditDeps) {
 
   const indexWisePermissionOptions = [
     {
-      label: 'Permission groups',
+      label: 'Index permission groups',
       options: actionGroups
         .filter((actionGroup) => actionGroup[1].type === 'index')
         .map((actionGroup) => actionGroup[0])
         .map(stringToComboBoxOption),
     },
+    {
+      label: 'Other permission groups',
+      options: actionGroups
```

Comment:
I'm not 100% sure if I understood the change correctly but I'll give it a shot and maybe you can clarify if I'm wrong?

All actionGroups that have type as `undefined` will be considered custom groups and will now be displayed in the dropdown list along with static actions groups?

If so can we change the backend to return type `custom` ? I understand that this might be out of scope of this fix however I think resolving that would be better IMO. `undefined` seems too broad. Thoughts?

Reply:
I think you are understanding the change correctly, but I feel like this is out of scope for this PR. We can consider it as a future feature enhancement/proposal, but not sure what is the benefit of changing `undefined` to custom? Even if it is not great/that is how it is today, and changing it I think would also involve handling this existing `undefined` case in case of migrations/previously stored custom groups, so not either way we have to make this change, wdyt?

Reply:
okay, sounds good. I was thinking maybe having the group type be custom to indicate custom group would be a good improvement. And I agree that is out of scope of this PR.